screens/chat-screen/home.js

```diff
@@ -20,10 +22,70 @@ const getID = () => Math.round(Math.random() * 1000000);
 
 export default class Home extends Component {
   state = {
-    messages: sleep_diary_messages,
+    messages: generic_messages,
     typingText: null,
-    isModalVisible: false
+    isModalVisible: false,
+    diary: '',
+    isLoading: true,
+    appState: new Set() 
   };
+  
+  async componentDidMount() {
+    
+    await this.isSleepDiaryEntered();
+    console.log('inside comp');
+    console.log(this.state.appState)
+
+    //determining message type
+    const { appState } = this.state;
+    if(appState.size > 0 ) {
+      if (appState.has(1)) {
+        this.setState({ messages: sleep_diary_messages });
+      }
+      this.setState({
+        isLoading: false,
+      });
+    }
+  }
+
+
+  /** 
+   * determines if there is a sleep diary entry for today
+   * sets app state to 1 if there is no entry
+   * sets app state to 2, 3, 4 if there is
+   */
+  isSleepDiaryEntered = async() => {
+    // var date = Moment(date).format("MM-DD-YYYY")
+    var appState = this.state.appState;
+    var date = "02-12-2020"; //for testing
```

Comment:
Nit: change `var` to `const`? It's very random

Reply:
is there a reason we prefer const over var?